docs/content/app-protect-dos/installation.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Installation with NGINX App Protect Dos
+description:
+weight: 1800
+doctypes: [""]
+toc: true
+---
+
+> **Note**: The NGINX Kubernetes Ingress Controller integration with NGINX App Protect requires the use of NGINX Plus.
+
+This document provides an overview of the steps required to use NGINX App Protect Dos with your NGINX Ingress Controller deployment. You can visit the linked documents to find additional information and instructions.
+
+## Prerequisites
+
+1. Make sure you have access to the Ingress controller image:
+    * For NGINX Plus Ingress controller, see [here](/nginx-ingress-controller/installation/pulling-ingress-controller-image) for details on how to pull the image from the F5 Docker registry.
+    * To pull from the F5 Container registry in your Kubernetes cluster, configure a docker registry secret using your JWT token from the MyF5 portal by following the instructions from [here](/nginx-ingress-controller/installation/using-the-jwt-token-docker-secret).
+    * It is also possible to build your own image and push it to your private Docker registry by following the instructions from [here](/nginx-ingress-controller/installation/building-ingress-controller-image).
+2. Clone the Ingress controller repo:
+    ```
+    $ git clone https://github.com/nginxinc/kubernetes-ingress/
+    $ cd kubernetes-ingress
+    $ git checkout v2.0.3
+    ```
+
+## Install the App Protect Dos Arbitrator
+
+- Deploy the app protect dos arbitrator
+    ```bash
+    kubectl apply -f deployment/appprotect-dos-arb.yaml
```

Comment:
The `nginx-ingress` namespace needs to be deployed first - starting with this throws an `Error from server (NotFound): error when creating "deployment/appprotect-dos-arb.yaml": namespaces "nginx-ingress" not found`

Reply:
I've updated the instructions to adda step to. add the namespace.